codewof/templates/users/user_detail.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}User: {{ user.get_full_name }}{% endblock %}
+
+{% block content %}
+  <div class="container">
+
+    <div class="row">
+      <div class="col-sm-12">
+        <h1>{{ user.get_full_name }}</h1>
+
+        <p><strong>Date joined</strong>: {{ user.date_joined }}</p>
+        <p><strong>Points earned</strong>: {{user.profile.points}}</p>
+        <b>Achievements:</b>
+        <br>
+        {% for badge in all_badges %}
+            {% if badge in user.profile.earned_badges.all %}
+              <img src="{% static badge.icon_name %}">
+              <u>{{badge.display_name}}</u>
+            {% else %}
+            <i>{{badge.display_name}}</i>
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/65398409-2c416300-de0b-11e9-8731-0d99764a00c4.png)
I think it would look nicer without all the non-achieved badges - or at least make them smaller, greyed out or something.
Also when words are underlined it looks like you should be able to click on them - in this case I could be expecting a longer description or some list of all badges I could earn

Reply:
I second this. When the page first loaded I thought they were badges I had already earnt. It could be cool to have a page to see all badges with the ones not earnt yet greyed out - like Alasdair has mentioned. 

Although I'm not sure if this is just a minimum viable product kinda thing where we will worry about this stuff later?

Reply:
Would be nice to have some padding on the badges so that they don't touch?

![image](https://user-images.githubusercontent.com/16066822/65400183-e9848880-de14-11e9-8fb0-defabbf5b3c1.png)